src/cmd/run.js

```diff
@@ -69,32 +87,83 @@ export default async function run(
 
   const manifestData = await getValidatedManifest(sourceDir);
 
-  const firefoxDesktopRunnerParams = {
+  const runners = [];
+
+  // Options shared between the created runners.
+  const commonRunnerParams = {
     // Common options.
     extensions: [{sourceDir, manifestData}],
     keepProfileChanges,
     startUrl,
     desktopNotifications,
+  };
 
-    // Firefox specific CLI options.
-    firefoxBinary: firefox,
-    profilePath: firefoxProfile,
-    customPrefs,
-    browserConsole,
-    preInstall,
+  if (!target || target.length === 0 || target.includes('firefox-desktop')) {
+    const firefoxDesktopRunnerParams = {
+      ...commonRunnerParams,
 
-    // Firefox runner injected dependencies.
-    firefoxApp,
-    firefoxClient,
-  };
+      // Firefox specific CLI options.
+      firefoxBinary: firefox,
+      profilePath: firefoxProfile,
+      customPrefs,
+      browserConsole,
+      preInstall,
+
+      // Firefox runner injected dependencies.
+      firefoxApp,
+      firefoxClient,
+    };
 
-  const firefoxDesktopRunner = new FirefoxDesktopExtensionRunner(
-    firefoxDesktopRunnerParams
-  );
+    const firefoxDesktopRunner = new FirefoxDesktopExtensionRunner(
+      firefoxDesktopRunnerParams
+    );
+
+    runners.push(firefoxDesktopRunner);
+  }
+
+  if (target && target.includes('firefox-android')) {
+    const firefoxAndroidRunnerParams = {
+      ...commonRunnerParams,
+
+      // Firefox specific CLI options.
+      profilePath: firefoxProfile,
+      customPrefs,
+      browserConsole,
+      preInstall,
+      firefoxApk,
+      adbDevice,
+      adbHost,
+      adbPort,
+      adbBinary,
+
+      // Injected dependencies.
+      firefoxApp,
+      firefoxClient,
+      adb: defaultADB,
+      desktopNotifications: defaultDesktopNotifications,
+      buildSourceDir: (extensionSourceDir: string) => {
+        return buildExtension({
```

Comment:
This call should add `showReadyMessage: false` so that it doesn't log an info message